src/shared/components/query/studyList/StudyList.tsx

```diff
@@ -31,6 +32,13 @@ export interface IStudyListProps {
     showSelectedStudiesOnly?: boolean;
 }
 
+const referenceGenomeBadgeColors: { [referenceGenome: string]: string } = {
```

Comment:
Lets put this in /shared/constants.ts file to encourage reuse elsewhere

Reply:
fixed in [765125b](https://github.com/cBioPortal/cbioportal-frontend/pull/4407/commits/765125bb1ba294e20e059cfe213bce3d7712325b)